examples/throughput.rs

```diff
@@ -52,8 +52,8 @@ impl FutureService for Server {
 const CHUNK_SIZE: u32 = 1 << 19;
 
 fn bench_tarpc(target: u64) {
-    let handle = Server.listen("localhost:0".first_socket_addr()).wait().unwrap();
```

Comment:
Yeah we should return a handle that has a way to gracefully shut down the service. I opened an #62.